web/packages/teleterm/src/ui/Search/useSearch.ts

```diff
@@ -174,7 +174,7 @@ export function useFilterSearch() {
   );
 }
 
-export function sortResults(
+export function sortAndLimitResults(
```

Comment:
Maybe `rankResults`? We could note in the JSDoc that it returns top 10 results. Or maybe we could even call it `getTopResults`.